MAUI/Autocomplete/UI-Customization.md

```diff
@@ -11,6 +11,10 @@ documentation: ug
 
 This section explains different UI customizations available in the [.NET MAUI Autocomplete](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfAutocomplete.html) control.
 
+Watch this video for a quick guide on customizing the .NET MAUI Autocomplete.
+
```

Comment:
@ManjudhanasekaranSF4622 , To get start quickly to customize the Appearance of .NET MAUI Autocomplete, you can check on this video:

Reply:
Sir, content revamped.